src/pages/bookmark-page/BookmarkPage.jsx

```diff
@@ -16,9 +16,9 @@ const BookmarkPage = () => {
 
   return (
     <main>
-      <p className="text-xl mt-2 font-bold">Bookmarked Posts</p>
-      {bookmarks.length === 0 && (
-        <p className="m-2 text-xl text-center">
+      <p className="text-xl mt-2 font-bold dark:text-white">Bookmarked Posts</p>
+      {bookmarkedPosts.length === 0 && (
```

Comment:
you can also write !bookmarkPosts.length . instead of comparing (bookmarkedPosts.length === 0)